### ISSUE TITLE
Strong name signing v5.8.2

### ISSUE THREAD

**Issue by michael-damatov** [meziantou/WPFFontAwesome#3]
The v5.8.2 is not strong-name signed anymore, so it cannot be referenced from the other strong-named assemblies.

**Comment by meziantou** [comment]
Thanks for reporting this issue. The attribute disappeared when I've added the support for .NET Core 3,0.
I've added a test, so it should be good now.

Could you test the version 5.8.2.1? (it will be published in a few minutes)

**Comment by michael-damatov** [comment]
The version 5.8.2.1 is not strong-named, however, icons are not shown.

`<fa:FontAwesomeIcon SolidIcon="Columns" />` simply doesn't work anymore.

**Comment by meziantou** [comment]
Ok, I think I've found the issue :) During the migration to the new toolset, I've removed the AssemblyInfo.cs file because the assembly attributes are now generated during the compilation. However in WPF, you also need the `ThemeInfo` attribute. This attribute is defined by default in `AssemblyInfo.cs`...

The Demo app in the project works because it includes the `Generic.xaml` file from the library manually. That's why I haven't found the issue before.

I hope #5 will fix the issue. The version 5.8.2.2 will be ready soon. I've also unlisted the packages that doesn't work from NuGet.
